File: cifrado/Escritorio/C#/CifradoCesarCshare/CifradoCesarCshare/Alfabeto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace CifradoCesarCshare
{
    class Alfabeto
    {
        public String caracteres;
        
        public Alfabeto(String letras)
        {
            caracteres = letras;
        }

        public int buscarPosicionLetra(char letra)
        {
            for (int i = 0; i < caracteres.Length; i++)
                if (caracteres[i] == letra)
                    return i;

            return -1;
        }

        public bool perteneceAlfabeto(char letra)
        {
            for (int i = 0; i < caracteres.Length; i++)
                if (caracteres[i] == letra)
                    return true;

            return false;
        }
        
    }
}

File: cifrado/Escritorio/C++/CifradoCesar/CifradoCesar/Alfabeto.cpp
#include "stdafx.h"
#include "Alfabeto.h"
#include <string>
using namespace std;



string caracteres;

Alfabeto::Alfabeto(string letras)
{
	caracteres = letras;
}

int Alfabeto::buscarPosicionLetra(char letra)
{
	for (int i = 0; i < caracteres.length(); i++)
		if (caracteres[i] == letra)
			return i;

	return -1;
}

bool Alfabeto::perteneceAlfabeto(char letra)
{
	for (int i = 0; i < caracteres.length(); i++)
		if (caracteres[i] == letra)
			return true;

	return false;
}